apps/api/src/domain/models/audio-item/entities/transcript.entity.ts

```diff
@@ -1,20 +1,34 @@
 import { NestedDataType } from '@coscrad/data-types';
+import { InternalError } from '../../../../lib/errors/InternalError';
+import { Maybe } from '../../../../lib/types/maybe';
+import { isNotFound, NotFound } from '../../../../lib/types/not-found';
+import { DeepPartial } from '../../../../types/DeepPartial';
 import { DTO } from '../../../../types/DTO';
+import { ResultOrError } from '../../../../types/ResultOrError';
 import BaseDomainModel from '../../BaseDomainModel';
+import {
+    DuplicateTranscriptParticipantError,
+    DuplicateTranscriptParticipantInitialsError,
+    DuplicateTranscriptParticipantNameError,
+} from '../errors';
 import { TranscriptItem } from './MediaTimeRange';
 import { TranscriptParticipant } from './transcript-participant';
 
 export class Transcript<T extends string = string> extends BaseDomainModel {
     // TODO Validate that there are not duplicate IDs here
```

Comment:
Do we do this yet?

Reply:
Linked to [this](https://www.pivotaltracker.com/story/show/184431225) relevant story.